cmd/manager/main.go

```diff
@@ -108,6 +106,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Define an kubeClient for any processes that need to run during operator startup or independent routines to use
```

Comment:
femtonit (also latent)
```suggestion
	// Define a kubeClient for any processes that need to run during operator startup or independent routines
```